sqlglot/dialects/tsql.py

```diff
@@ -29,6 +96,10 @@ class Tokenizer(Tokenizer):
             "UNIQUEIDENTIFIER": TokenType.UNIQUEIDENTIFIER,
             "XML": TokenType.XML,
             "SQL_VARIANT": TokenType.VARIANT,
+            "NVARCHAR(MAX)": TokenType.TEXT,
+            "VARCHAR(MAX)": TokenType.TEXT,
+            "NVARCHAR": TokenType.VARCHAR,
+            "NCHAR": TokenType.CHAR,
```

Comment:
Why are these needed?

Reply:
Thanks for the headsup. THe last two tokens are indeed not needed and i removed them now.